python/ml4ir/base/config/default_evaluation_config.yaml

```diff
@@ -0,0 +1,22 @@
+power_analysis:
```

Comment:
@mohazahran Oh wow, I'm glad you did this. I thought you were planning to do it in a follow up work/PR.